client/state/ui/action-log/reducer.js

```diff
@@ -12,22 +11,33 @@ import {
 	GUIDED_TOUR_UPDATE,
 	THEMES_RECEIVE,
 	PREVIEW_IS_SHOWING,
-	ROUTE_SET,
 } from 'state/action-types';
 
-const isRelevantActionType = includes.bind( null, [
+const isDesignRoute = ( { path } ) => 0 === path.indexOf( '/design' );
```

Comment:
Good question. So we either disable this optimization, or if we really care we extend it to also log Stats routes.